Interface.H

```diff
@@ -21,7 +21,7 @@ protected:
     std::string meshName_;
 
     //- Point locations type (faceCenters vs faceNodes)
-    std::string locationsType_;
+    LocationType locationsType_ = LocationType::none;
```

Comment:
Is there a difference?

Reply:
Not really, I just like the explicit initialization more, especially since it can save us from implicit conversions.
But the rest of the code mostly/only uses assignment anyway.

Reply:
Done